content/data-reporting/reports-api/index.mdx

```diff
@@ -200,6 +201,82 @@ group_by=hour" \
 
 <Text>
 
+# Chat Missed
```

Comment:
```suggestion
# Chats Missed
```

Reply:
why not Missed chats?

Reply:
My mistake with a copy...fixed in https://github.com/livechat/livechat-public-docs/pull/572/commits/72db3c1bd3011dd7ce571afc3f608468d0a8b343